File: youlexuan-shop-web/src/main/java/com/offcn/shop/controller/loginController.java
package com.offcn.shop.controller;

import com.alibaba.dubbo.config.annotation.Reference;
import com.offcn.seller.service.SellerService;
import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;

@RestController
public class loginController {

    @Reference
    private SellerService sellerService;

    @RequestMapping("/showSellerName")
    public Map showSellerName(){

        String name = SecurityContextHolder.getContext().getAuthentication().getName();
        Map map=new HashMap();
        map.put("name",name);
        return map;
    }

    @RequestMapping("/showSellerTime")
    public Map showSellerTime(){

        String name = SecurityContextHolder.getContext().getAuthentication().getName();
        return sellerService.showSellerTime(name);
    }

    @RequestMapping("/setLoginTime")
    public void setLoginTime(){

        String name = SecurityContextHolder.getContext().getAuthentication().getName();
        sellerService.setLoginTime(name);
    }

}

File: youlexuan-shop-web/src/main/webapp/js/controller/loginController.js
app.controller('loginController',function ($scope,$http) {

    //设置登录名
    $scope.showSellerName=function(){
        $http.post("/showSellerName.do").success(function (map) {

            $scope.name=map.name;
        })
    };
    //设置最后登录时间
    $scope.showSellerTime=function(){
        $http.post("/showSellerTime.do").success(function (map) {

            $scope.time=map.time;
        })
    };
    //设置登录时间
    $scope.setLoginTime=function () {

        $http.post("/setLoginTime.do");
    }
});